docs/health.md

```diff
@@ -18,3 +18,70 @@ with at least one value for `hostname` or `IP`.
 
 ### PersistentVolumeClaim
 * The `status.phase` is `Bound`
+
+## Health Checks Customization
```

Comment:
I like the "how to" format for documentation?

E.g. "How to customize your health check"